docs/reST/ref/context.rst

```diff
@@ -0,0 +1,52 @@
+.. include:: common.txt
+
+:mod:`pygame.context`
+======================
+
+.. module:: pygame.context
+    :synopsis: pygame module to provide additional context about the system
+
+| :sl:`pygame module to provide additional context on the system`
+
+**EXPERIMENTAL!** This API may change or disappear in later pygame releases. 
+If you use this, your code may break with the next pygame release.
+This is a new module, so we are marking it experimental for now.
+We probably won't have to change API, but we're keeping the possibility
+open just in case something obvious comes up.
+
+.. versionadded:: 2.1.3
+
+.. function:: get_pref_path
+
+   | :sl:`get a writeable folder for your app`
+   | :sg:`get_pref_path(org, app) -> path`
+
+   When distributing apps, it's helpful to have a way to get a writeable path,
+   because it's what apps are expected to do, and because sometimes the local
+   space around the app isn't writeable to the app.
+
+   This function returns a platform specific path for your app to store
+   savegames, settings, and the like. This path is unique per user and
+   per app name.
+
+   It takes two strings, ``org`` and ``app``. It then will figure out the
```

Comment:
I think there should be a little bit more explanation of what the two strings are supposed to represent with an example.

i.e. 'org' is for the organisation who developed/published the application and 'app' is for the name of the application e.g. 'Valve' is the developer, or 'org' behind the 'app' called 'Half-life'.